src/libraries/math/SafeMathUint256.sol

```diff
@@ -45,4 +44,22 @@ library SafeMathUint256 {
             return b;
         }
     }
+
+    function safeToMultiply(uint256 a, uint256 b) internal constant returns (bool) {
```

Comment:
Curious: what made you decide to extract this out into a separate method rather than inlining into `fxpMul`?

Reply:
I did it because the mul() and fxpMul() functions were both using this same logic, and this is how it was being done in safeMath.sem.  I realize it's kind of awkward, though.  I'd be ok with putting it inline in these functions instead.